api/v1/GameSession.php

```diff
@@ -79,7 +79,7 @@ public static function getRequestApiRootAsync(): PromiseInterface
 
         $_SERVER['HTTPS'] ??= 'off';
         /** @noinspection HttpUrlsUsage */
-        $protocol = ($_SERVER['HTTPS'] == 'on') ? "https://" : "http://";
+        $protocol = ($_SERVER['HTTPS'] == 'on') ? "https://" : ($_ENV['URL_WEB_SERVER_SCHEME'] ?? "http://");
```

Comment:
@JacopoFabrini since the websocket server is a console app, not a web server app , it does not have any $_SERVER variables. so it has to rely on the env variable URL_WEB_SERVER_SCHEME for choosing the correct scheme.

Reply:
Thanks for the explanation!